src/gemdat/jumps.py

```diff
@@ -227,13 +227,16 @@ def collective(self, max_dist: float = 1) -> Collective:
 
     @weak_lru_cache()
     def activation_energies(
-            self) -> dict[tuple[str, str], tuple[float, float]]:
+            self,
+            n_parts: int = 10) -> dict[tuple[str, str], tuple[float, float]]:
```

Comment:
Jumps.activation_energies might be better returned as pandas DataFrame, feel free to make an issue from this

Reply:
Yep, see #249